resources/views/auth/login.blade.php

```diff
@@ -36,7 +36,6 @@ class="flex flex-col p-8 mx-4 border rounded-lg border-theme-secondary-200 md:mx
                     <div class="mb-4">
                         <div class="flex flex-1">
                             <x-ark-input
-                                type="email"
```

Comment:
why was this removed? 🤔

Reply:
@ItsANameToo The login form accepts email or username, event the label says that currently was only accepting an email

Reply:
ah nice catch, probably an inconsistency between deployer and marketsquare that caused that 👍